src/App.js

```diff
@@ -9,20 +11,61 @@ import Tech from "./screens/Tech";
 import Projects from "./screens/Projects";
 import AboutMe from "./screens/AboutMe";
 import Contact from "./screens/Contact";
-import { useState } from "react";
+
+import { isInViewPort } from "./screens/lib/responsiveHelpers";
 
 function App() {
+  let welcome;
+  let cv;
+  let tech;
+  let projects;
+  let about;
+  let contact;
+
   const [active, setActive] = useState(null);
 
   const handleActive = (item) => {
     setActive(item);
   };
+
+  window.addEventListener("DOMContentLoaded", () => {
+    console.log("DOM fully loaded and parsed");
+    welcome = document.getElementById("welcome");
+    cv = document.getElementById("cv");
+    tech = document.getElementById("tech");
+    projects = document.getElementById("projects");
+    about = document.getElementById("about");
+    contact = document.getElementById("contact");
+  });
+
```

Comment:
This one freaked me out a bit ... isn't there a nicer way to declare the variables?